src/components/GweiPopup/GweiPopup.js

```diff
@@ -223,6 +226,12 @@ const InputRow = styled.div`
 	display: flex;
 `;
 
+const GlobalStyle = createGlobalStyle`
+	.rc-slider-tooltip {
+		z-index: ${Z_INDEX.MODAL};
```

Comment:
giving it the same z-index of the modal, fixes the problem.

Reply:
cheers!